src/asmcnc/apps/maintenance_app/widget_maintenance_brush_save.py

```diff
@@ -87,9 +87,23 @@ def save(self):
 
         # TIME FOR DATA VALIDATION
 
-        # Read from screen and convert hours into seconds
-        use = float(self.sm.get_screen('maintenance').brush_use_widget.brush_use.text)*3600
-        lifetime = float(self.sm.get_screen('maintenance').brush_life_widget.brush_life.text)*3600 
+        # Read from screen
+        use_str = self.sm.get_screen('maintenance').brush_use_widget.brush_use.text
+        lifetime_str = self.sm.get_screen('maintenance').brush_life_widget.brush_life.text
+
+        if use_str == '' or lifetime_str == '':
```

Comment:
rather than using an "if" here, use a try-except wrapped around the conversion. 

I managed to crash it by putting in a '-'.